hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/rollback/TestMergeOnReadRollbackActionExecutor.java

```diff
@@ -126,21 +126,19 @@ public void testMergeOnReadRollbackActionExecutor(boolean isUsingMarkers) throws
     for (Map.Entry<String, HoodieRollbackPartitionMetadata> entry : rollbackMetadata.entrySet()) {
       HoodieRollbackPartitionMetadata meta = entry.getValue();
       assertEquals(0, meta.getFailedDeleteFiles().size());
-      assertEquals(0, meta.getSuccessDeleteFiles().size());
+      assertEquals(1, meta.getSuccessDeleteFiles().size());
```

Comment:
not following the logic here.. why should there be 1 successful delete at this stage?

Reply:
Because both list and marker based rollback for MOR can not delete the intermediate files correctly.

Reply:
So does it need fixing?

Reply:
No, the test looks good.